### PR TITLE
Various small documentation tweaks

### PR DESCRIPTION
**Pull request by gregrs-uk**
* Added full stops and capital letters for consistency.
* Fixed some typos and grammatical/punctuation errors.
* Added backticks for `code formatting`.

### CODE REVIEW

File: R/geom-point.r [hunk 1]
```diff
@@ -12,7 +12,7 @@
 #' another variable is mapped to point shape or colour, however.
 #'
 #' @section Overplotting:
-#' The biggest potential problem with a scatterplot is overplotting: whenever
+#' The biggest potential problem with a scatterplot is overplotting; whenever
```
Comment (hadley) [inline R/geom-point.r]: `:` seems correct to me here

Comment (hadley) [reply R/geom-point.r]: In fact, can you revert all your colon changes?

File: R/geom-path.r [hunk 2]
```diff
@@ -6,16 +6,16 @@
 #' when changes occur. The `group` aesthetic determines which cases are
 #' connected together.
 #'
-#' An alternative parameterisation is [geom_segment()]: each line
+#' An alternative parameterisation is [geom_segment()]; each line
```
[the remark this answers is not in the document]
Comment (gregrs-uk) [reply R/geom-path.r]: I agree. Done.

**Comment by gregrs-uk** [comment]
Thanks. I've reverted cases where I had changed a colon to a semicolon.

I've left the change to `R/aes-colour-fill-alpha.r` line 4 (sorry, not sure how to provide a link) because I believe the previous semicolon was incorrect.

**Comment by gregrs-uk** [comment]
I tried to resolve the apparently conflicting files but it doesn't seem to have worked, apologies.

**Comment by hadley** [comment]
This is lovely, thanks for all your hard work on this!

**Comment by gregrs-uk** [comment]
No problem, thanks!

**Pull request merged**